models/people.py

```diff
@@ -0,0 +1,47 @@
+class Person(object):
+    """Models the kind of people available at Andela,
+    It forms the base class from which classes Fellow and Staff inherit"""
+
+    person_id = 0
```

Comment:
The initialization of **person_id** is necessary but I have deleted the **fellow_id** and **staff_id**